extensions/interactions/LogicProof/static/js/complete.spec.ts

```diff
@@ -66,6 +66,7 @@ describe('Full system', function() {
   it('should reject proofs with any error', function() {
     expect(function() {
       completeCheck('p', 'p', 'we knew p');
+      // eslint-disable-next-line oppia/no-incomplete-throw
```

Comment:
Do not use eslint-disable here or anywhere else, it is not recommended to use disable statements because there won't be any benefit of adding these lint checks if we are ultimately going to use disable statements.

Reply:
Using eslint-disable here because this check was diabled in some of the files ([ref](https://github.com/oppia/oppia/pull/13252/files#diff-2ec95fb28a886d8e7cf26b726170551779907cbe7dde6bdd0f268076d70f4682L154))
is it ok to use eslint-disable in those files or should I add those files to excluded file for the check in eslintrc.

Reply:
Yes, add these files as excluded files in eslintrc.

Reply:
Done!